docs/docs-content/clusters/edge/cluster-management/certificate-renewal.md

```diff
@@ -0,0 +1,126 @@
+---
+sidebar_label: "Renew Certificates for Air-gap Clusters"
+title: "Renew Certificates for Air-gap Clusters"
+description: "Learn how to renew certificates for different Kubernetes components in your cluster."
+hide_table_of_contents: false
+sidebar_position: 80
+tags: ["edge", "architecture"]
+---
+
+Kubernetes uses certificates to secure the communication between different components of a cluster. Using these
+certificates allows Kubernetes to secure API connections, verify the authenticity of the nodes, and encrypt connections.
+All certificates have an expiry date, and need to be renewed periodically.
+
+This page guides you through the different methods used to renew certificates in an air-gapped Palette Edge cluster. An
+air-gapped cluster means a cluster that has no connection to a Palette instance.
+
+## Limitations
+
+- The procedure described in this guide only renews certificates for control plane nodes. Certificate renewal for worker
+  nodes is not supported.
+- The certificates used are generated by Kubernetes. You cannot use your own certificates.
+
+## Automatic Renewal
+
+Palette Edge will automatically renew all certificates used by your cluster for you 30 days before they expire. You can
+follow the steps below to check when the next automatic renewal will happen.
```

Comment:
```suggestion
Palette Edge will automatically renew all control plane certificates your cluster uses for you 30 days before they expire. You can
follow the steps below to check when the next automatic renewal will happen.
```

Simplified the sentence and clarified what "all" means to not generate confusion.